listenbrainz/webserver/static/css/follow.less

```diff
@@ -0,0 +1,50 @@
+@purple: #353070;
+@white: #ffffff;
+
+#follow-button {
+  margin-left: 10px;
+  font-family: "Sintony";
+  color: @white;
+  background: @purple;
+  border: 2px solid @purple;
+  font-weight: bold;
+  padding: 5px 8px;
+  border-radius: 4px;
+  transition: color 0.42s, background 0.42s;
+  text-align: center;
+
+
+  &.icon-only {
+    > .text {
+      width: 0px;
+      margin-left: 0px;
+      display: inherit;
+      vertical-align: inherit;
+      transition: width 0.42s, margin-left 0.42s;
+      overflow-x: hidden;
+    }
+
+    &:hover, &:focus {
+      background: white;
+      color: #353070;
+
+      > .text {
+        min-width: 4.75em;
```

Comment:
This should probably be `width` instead, since it is the property used in the `transition` (on line 23).
Otherwise you get a sort of jumpy animation.

Reply:
done